api/src/main/java/io/smallrye/stork/api/config/ServiceDiscoveryName.java

```diff
@@ -0,0 +1,13 @@
+package io.smallrye.stork.api.config;
+
+import static java.lang.annotation.ElementType.TYPE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(TYPE)
+public @interface ServiceDiscoveryName {
```

Comment:
the problem is that the interface name and the annotation name would be the same then and, aside from being confusing, it would mean that one needs to use the fully qualified name for the import